sequencer/src/genesis.rs

```diff
@@ -74,6 +76,25 @@ impl Genesis {
     }
 }
 
+impl Genesis {
+    pub async fn validate_contract(&self, l1_rpc_url: String) -> anyhow::Result<()> {
+        let provider = Provider::<Http>::try_from(l1_rpc_url)?;
+        let accounts_vec: Vec<_> = self.accounts.keys().collect();
```

Comment:
These accounts are prefunded and used for testing purposes such as for builder to have enough funds to build blocks. the fee contract address is in `chain_config`.
So, we should validate` self.chain_config.fee_contract` as  well as fee_contract in chain config of all the upgrades

Reply:
ah thank you https://github.com/EspressoSystems/espresso-sequencer/pull/2000/commits/50d4b9c46e5d29949af3d6a40851919d8c0c1362